third_party/txt/src/minikin/FontFamily.cpp

```diff
@@ -146,15 +146,27 @@ static FontFakery computeFakery(FontStyle wanted, FontStyle actual) {
   return FontFakery(isFakeBold, isFakeItalic);
 }
 
-FakedFont FontFamily::getClosestMatch(FontStyle style) const {
+FakedFont FontFamily::getClosestMatch(FontStyle style,
+                                      uint32_t codepoint,
+                                      uint32_t variationSelector) const {
```

Comment:
I recommend this for readability, but you can ignore it.

```suggestion
                                      uint32_t codepoint /* = 0 */,
                                      uint32_t variationSelector /* = 0 */) const {
```